packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -478,6 +478,7 @@ class FlutterCommandRunner extends CommandRunner<void> {
   }
 
   static void initFlutterRoot() {
+    Cache.checkPermissions = true;
```

Comment:
It looks like initFlutterRoot is only called in one place from analyze_continuously_test.dart. Two questions:
1. Comments above indicate that permissions should not be checked for tests, so shouldn't this not work?
2. Can the test in analyze_continuously_test.dart be rewritten to avoid the need for this function?

Reply:
My bad, I was working on this too late and didn't clean it up :(